packages/node-opcua-server/source/opcua_server.ts

```diff
@@ -262,9 +262,9 @@ async function _attempt_to_close_some_old_unactivated_session(server: OPCUAServe
 
 function getRequiredEndpointInfo(endpoint: EndpointDescription) {
     assert(endpoint instanceof EndpointDescription);
-    // It is recommended that Servers only include the server.applicationUri,  endpointUrl, securityMode,
-    // securityPolicyUri, userIdentityTokens, transportProfileUri and securityLevel with all
-    // other parameters set to null. Only the recommended parameters shall be verified by
+    // It is recommended that Servers only include the server.applicationUri, server.applicationType, server.applicationName,
```

Comment:
This sentences come from the OPCA spec ( probably v102). 
Now that the spec are available online we could add this link to the comment for cross reference : https://reference.opcfoundation.org/v104/Core/docs/Part4/5.6.2/

> List of Endpoints that the Server supports.
The Server shall return a set of EndpointDescriptions available for the serverUri specified in the request. The EndpointDescription type is defined in [7.10](https://reference.opcfoundation.org/v104/Core/docs/Part4/7.10). The Client shall verify this list with the list from a DiscoveryEndpoint if it used a DiscoveryEndpoint to fetch the EndpointDescriptions.
It is recommended that Servers only include the server.applicationUri, endpointUrl, securityMode, securityPolicyUri, userIdentityTokens, transportProfileUri and securityLevel with all other parameters set to null. Only the recommended parameters shall be verified by the client.


https://reference.opcfoundation.org/v105/Core/docs/Part4/5.6.2/
>List of Endpoints that the Server supports.  
> The Server shall return a set of EndpointDescriptions available for the serverUri specified in the request. All Endpoints are returned if the serverUri is null or empty. The EndpointDescription type is defined in [7.14](https://reference.opcfoundation.org/v105/Core/docs/Part4/7.14). The Client shall verify this list with the list from a DiscoveryEndpoint if it used a DiscoveryEndpoint to fetch the EndpointDescriptions.
It is recommended that Servers only include the server.applicationUri, endpointUrl, securityMode, securityPolicyUri, userIdentityTokens, transportProfileUri and securityLevel with all other parameters set to null or empty. Only the recommended parameters shall be verified by the Client.

Reply:
Done @erossignon , What do you think?